deploys/demo.syncfor.science/config/vendors/Allscripts.yml

```diff
@@ -1,43 +1,49 @@
 ---
 api:
-  url: https://apis-test.followmyhealth.com/fhir-uat/api/
-  patient: 3dcfa065-5a1e-4577-aa04-a719013f736b
+  versions:
```

Comment:
Are there other keys that can appear here besides `versions`?  I don't mind the extra node in the tree, but want to make sure I know what's happening.

Reply:
There are not any other keys yet, I do like wordy yml. Happy to change if it gets out of hand.